LocalSettings.php

```diff
@@ -2593,13 +2593,6 @@
 				'oauth-enable' => true,
 				'managewiki' => true,
 				'managewiki-restricted' => true,
-			),
```

Comment:
the build fails because you removed this

Reply:
Oh pfft